databases/week1/Assignment_Task_Queries.sql

```diff
@@ -0,0 +1,85 @@
+USE database_week1;
+-- Find out how many tasks are in the task table
+SELECT 
+    `title`
+FROM
+    `task`;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+    `due_date`
+FROM
+    `task`
+WHERE
+    `due_date` IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT 
+    task.id, task.title, status.name
+FROM
+    `task`
+        INNER JOIN
+    `status` ON status.id = task.status_id
+WHERE
+    status.name = 'Done';
+
+-- Find all the tasks that are not marked as done
+SELECT 
+    task.id, task.title, status.name
+FROM
+    `task`
+        INNER JOIN
+    `status` ON status.id = task.status_id
+WHERE
+    status.name != 'Done';
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT 
+    id, title, created
+FROM
+    `task`
+WHERE
+    `created`
+ORDER BY `created` DESC;
+
+-- Get the single most recently created task
+SELECT 
+    id, title, MAX(created)
+FROM
+    `task`;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT 
+    title, description
```

Comment:
Here you are asked to return the title & due_date of all tasks that contains "database" in their title or description..
Something like this
```
SELECT
    title, due_date
FROM
    `task`
WHERE
    title LIKE '%database%'
        OR description LIKE '%database%';
```